upgrade/upgrade-1.3.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+* 2007-2015 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Academic Free License (AFL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/afl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    Línea Gráfica E.C.E. S.L.- https://www.lineagrafica.es/ <soporte@lineagrafica.es>
+* @copyright 2007-2015 TELEFONICA SERVICIOS INTEGRALES DE DISTRIBUCIÓN S.A.U.
+* @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+* International Registered Trademark & Property of TELEFONICA SERVICIOS INTEGRALES DE DISTRIBUCIÓN S.A.U.
+*/
+
+if (!defined('_PS_VERSION_'))
+	exit;
+
+function upgrade_module_1_3($module)
+{
+	$configs = DB::getInstance()->executeS('SELECT * FROM '._DB_PREFIX_.'configuration WHERE `name` LIKE "%ZELERIS%"');
+	if (is_array($configs))
+		foreach($configs as $config)
+			$config_installed[] = $config['name'];
+	
+	if (in_array('ZELERIS_DESCRIPCION_MERCANCIA',$config_installed))
+	{
+		$sql='UPDATE '._DB_PREFIX_.'configuration SET `name` = "ZELERIS_MERCHANDISE_DESCRIPTION" WHERE `name`="ZELERIS_DESCRIPCION_MERCANCIA"';
```

Comment:
You should update your data like this (assuming you want to remove the old fields:

```php
Configuration::updateValue('ZELERIS_MERCHANDISE_DESCRIPTION', Configuration::get('ZELERIS_DESCRIPCION_MERCANCIA'));
Configuration::deleteByName('ZELERIS_DESCRIPCION_MERCANCIA');
```